test/cases/translate/unnamed_child_types_of_typedef_receive_typedef's_name.c

```diff
@@ -7,12 +7,11 @@ typedef struct {
 } Bar;
 
 // translate
-// expect=fail
 //
 // pub const FooA: c_int = 0;
 // pub const FooB: c_int = 1;
-// pub const Foo = c_uint;
```

Comment:
Hmm, these `pub`s should probably still be there... Looking into it.

Reply:
I don't understand the test and I didn't modify type generation in this PR so I'm going to disable this test again for now.